### ISSUE TITLE
Issue with Legrand 067776 wired shutter switch

### ISSUE THREAD

**Issue by jd1900** [Koenkk/zigbee-herdsman-converters#2339]
I've sent a pull request (https://github.com/Koenkk/zigbee-herdsman-converters/pull/2338) to fix the inverted working state.

But still, it isn't working fine.

For instance, I have this blind half open, however, I can't choose open or close, it only gets the last value when fully opened or fully closed. In this case, it was fully closed. After that, in HA only open is possible:
![image](https://user-images.githubusercontent.com/63632981/110363768-4264d280-8043-11eb-9573-e0d48633aeaf.png)

This is fine. But I then move it to the middle:
Debug Received MQTT message on 'zigbee2mqtt/PERSIANA_06_salon_peque/set' with data 'OPEN'
Debug Received MQTT message on 'zigbee2mqtt/PERSIANA_06_salon_peque/set' with data 'STOP'

However, it remains the same way, not able to close it even being in the middle.
But the Zigbee2Mqtt always exposes the OPEN and CLOSE commands, but no the current position and not the STOP option:
![image](https://user-images.githubusercontent.com/63632981/110364012-8fe13f80-8043-11eb-9284-a8fa77b9ba21.png)
It would be nice to get the position (as it is reported by the device) and allow the STOP function.

I guess this is a problem from the device.
After a while, I get this message:
Info MQTT publish: topic 'zigbee2mqtt/PERSIANA_06_salon_peque', payload '{"last_seen":"2021-03-08T18:23:48.359Z","linkquality":87,"position":0,"update":{"state":"idle"},"update_available":false}'

However, this is false. As the position is in the middle (maybe 50 would be better option). 

And, as another comment, it takes quite a bit to update once it is fully opened or closed. I'd say it worked fine with the report option. But not now. It updates the position value, but after quite a few minutes usually.
This are the configs of this device:
![image](https://user-images.githubusercontent.com/63632981/110364254-d59e0800-8043-11eb-811d-7f2726c63475.png)

![image](https://user-images.githubusercontent.com/63632981/110364447-1138d200-8044-11eb-8c22-00c081603deb.png)

When joining the device it tries to add reporting for the device but it fails. Same in all my shutter switches. All of them in the latest firmware from Legrand.
I've tried to add reporting for other attributes, and I can get this working:
![image](https://user-images.githubusercontent.com/63632981/110364836-94f2be80-8044-11eb-90d1-5f3e9481cbe3.png)
Which is not the attribute tried to add in the config.
But it doesn't seem to improve anything.

It seems the problem has to be with the way that the commands are sent.
As described here: 
https://github.com/Koenkk/zigbee-herdsman-converters/pull/2338
{"position": 100} fully opens the cover, and inmediately let to close the cover. Reporting the value to 100.
{"position": 1-99} closes the cover but the it state to that value, being able to both open or close the cover. The reported value is inmediatley the chosen.
{"position": 0} closes the cover and set it value to closed, not being able to press "close" in the GUI.

For instance, when set to 100, after that, I can close the shutter.
When set to 30:
![image](https://user-images.githubusercontent.com/63632981/110366268-56f69a00-8046-11eb-9e08-fd840dadccda.png)
I can close or open, despite being closed.

When set to 0:
![image](https://user-images.githubusercontent.com/63632981/110366306-64ac1f80-8046-11eb-8527-cca44a214ee6.png)
I can only open. Being close.

However, in the Z2M GUI it is never reported the position value:
![image](https://user-images.githubusercontent.com/63632981/110366551-bbb1f480-8046-11eb-9a08-448cb7fa08e4.png)

I guess there is some problem regarding the way OPEN and CLOSE commands are sent (both in Zigbee2MQTT GUI and HA GUI), as they are not equivalent to position 100 and position 0.
![image](https://user-images.githubusercontent.com/63632981/110367290-c3be6400-8047-11eb-9b96-58ff4233c564.png)

So, to recap:
- Add support for STOP in the Z2M GUI.
- Add support for the reported position in the Z2M GUI. 
- Be able to always set STOP, OPEN and CLOSE in HA GUI regardless of the position, since it is very slow and unreliable.
- Be equivalent to position 0 to close and position 100 to open.

As the device doesn't report fine the OPEN and CLOSE values or its position, I guess this would be the best option.

**Comment by Lionel642** [comment]
Hi; sorry, maybe not the correct place, just got these Legrand roller shutter controllers, paired with my zigbee2mqtt, I can close/open/stop through mqtt or Z2M UI (except stop, stop button is missing I guess)).

But cannot set the position, using mqtt, or the Z2M UI, position 0 or 100 fully open or fully close, ok, but with intermediate numbers, nothing happens. Got it to work on your side? (@jd1900 as you seems to have the same device)

Thanks.
Lionel

**Comment by jd1900** [comment]
> But cannot set the position, using mqtt, or the Z2M UI, position 0 or 100 fully open or fully close, ok, but with intermediate numbers, nothing happens. Got it to work on your side? (@jd1900 as you seems to have the same device)

It is not implemented in the original firmware, that's why you can't do it.
Intermediate are as 0 if I recall fine (or maybe as 100). They work, but not as actual intermediate.
